plan/plan.go

```diff
@@ -263,6 +270,16 @@ func resolveComponentCommon(commons ...v2.Common) ComponentCommon {
 		}
 	}
 
+	var snowflakePlan *SnowflakeProvider
+	snowflakeConfig := v2.ResolveSnowflakeProvider(commons...)
+	if snowflakeConfig != nil {
+		snowflakePlan = &SnowflakeProvider{
+			Account: *snowflakeConfig.Account,
```

Comment:
This assumes if one is present then all are, but resolve doesnt

Reply:
The trick here is that validation should assure that this is the case. And the rule is that you can only run the plan after validation. That said, there might be bugs...